### PR TITLE
MovieTopInfo API integrated

### PR DESCRIPTION
**Pull request by Mateusz-95**
I'm adding a pull request with the MovieTopInfo component integrated with the API. I'm not adding toFixed to the movie rating, as I saw that it was already added in a recent pull request by Maciej. I don't want to duplicate.

### CODE REVIEW

File: src/features/MovieDetails/index.js [hunk 1]
```diff
@@ -1,12 +1,32 @@
...
+  const status = useSelector(selectStatus);
+  const dispatch = useDispatch();
+  const { id } = useParams();
+  const { title, backdrop_path, vote_average, vote_count } =
+    useSelector(selectDetails) ?? {};
+
+  useEffect(() => {
+    dispatch(getMovieId(id));
+  }, [status, dispatch, id]);
+
```
Comment (lenarmathias) [inline src/features/MovieDetails/index.js:20]: There will probably be a need to add another useEffect to clear the API id in future updates.

File: src/features/getData.js [hunk 2]
```diff
@@ -11,6 +11,9 @@ export const getMovieDetails = async ({ movieId }) => {
   return await response.data;
 };
 
+const movie = await getMovieDetails({ movieId: 926393 });
+console.log(movie.title);
+
```
Comment (lenarmathias) [inline src/features/getData.js]: I think you forgot to delete that after testing.

**Review by lenarmathias — APPROVED** [review]
I think everything is good to go. Just check my comment on getData.js.

Good job 👍

File: src/features/MovieDetails/index.js (hunk 1 above)
Comment (Mateusz-95) [reply src/features/MovieDetails/index.js:20]: For now, everything displays correctly, so I don't think it's necessary. If there's a need to add a cleanup function in the future, we can always add it then

File: src/features/getData.js (hunk 2 above)
Comment (Mateusz-95) [reply src/features/getData.js]: Yes, thank you ^^

**Pull request merged**